packages/xo-server/config.toml

```
@@ -134,6 +134,8 @@ namespace = 'xoProxyAppliance'
 
 proxyName = 'Proxy {date}'
 
+proxyProductId = 'xo-proxy'
```

What do you think of?

```suggestion
licenseProductId = 'xo-proxy'
```